docs/setup/install.md

```diff
@@ -0,0 +1,29 @@
+# Installation
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
+
+Kueue **Requires Kubernetes 1.22 or newer**. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
+Kubernetes 1.21 can be used if the SuspendJob [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled
+
+## Deploy Kueue
+
+You can deploy Kueue in your cluster with the following command:
+
+```sh
+IMAGE_REGISTRY=registry.example.com/my-user make image-build image-push deploy
+```
+
+The controller will run in the `kueue-system` namespace.
+
+## Undeploy Kueue
```

Comment:
Could it be accurate to say "Uninstall" instead?

Undeploy doesn't sound common.